spec/easy_post/address_spec.rb

```diff
@@ -18,7 +18,7 @@
         allow(EasyPost).to receive(:make_request).and_return({})
         expect {
           described_class.create_and_verify(ADDRESS[:california])
-        }.to raise_error EasyPost::Error, /Unable to verify addres/
```

Comment:
Your copy of this lib locally is super outdated as these tests are long gone haha. You'll want to fetch from origin and rebase before moving on.

Reply:
Woof.